luarules/gadgets/pve_supply_drops.lua

```diff
@@ -1,12 +1,12 @@
 local teams = Spring.GetTeamList()
 mapsizeX = Game.mapSizeX
 mapsizeZ = Game.mapSizeZ
+local scavengerAITeamID = Spring.Utilities.GetScavTeamID()
+local scavengerAllyTeamID = Spring.Utilities.GetScavAllyTeamID()
 for i = 1,#teams do
 	local luaAI = Spring.GetTeamLuaAI(teams[i])
 	if luaAI and luaAI ~= "" and string.sub(luaAI, 1, 12) == 'ScavengersAI' then
 		scavengersAIEnabled = true
```

Comment:
Use `Spring.Utilities.Gametype.IsScavengers()` for determining whether Scavengers are enabled.

Reply:
lol forgot that existed

Reply:
fixed